test/Npgsql.Tests/CommandTests.cs

```diff
@@ -158,18 +158,37 @@ public async Task Timeout()
 
             // Mono throws a socket exception with WouldBlock instead of TimedOut (see #1330)
             var isMono = Type.GetType("Mono.Runtime") != null;
-            using (var conn = await OpenConnectionAsync(ConnectionString + ";CommandTimeout=1"))
-            using (var cmd = CreateSleepCommand(conn, 10))
-            {
-                Assert.That(() => cmd.ExecuteNonQuery(), Throws.Exception
-                    .TypeOf<NpgsqlException>()
-                    .With.InnerException.TypeOf<IOException>()
-                    .With.InnerException.InnerException.TypeOf<SocketException>()
-                    .With.InnerException.InnerException.Property(nameof(SocketException.SocketErrorCode)).EqualTo(isMono ? SocketError.WouldBlock : SocketError.TimedOut)
-                    );
-                Assert.That(conn.FullState, Is.EqualTo(ConnectionState.Broken));
-            }
+            using var conn = await OpenConnectionAsync(ConnectionString + ";CommandTimeout=1");
+            using var cmd = CreateSleepCommand(conn, 10);
+            Assert.That(() => cmd.ExecuteNonQuery(), Throws.Exception
+                .TypeOf<NpgsqlException>()
+                .With.InnerException.TypeOf<IOException>()
+                .With.InnerException.InnerException.TypeOf<SocketException>()
+                .With.InnerException.InnerException.Property(nameof(SocketException.SocketErrorCode)).EqualTo(isMono ? SocketError.WouldBlock : SocketError.TimedOut)
+                );
+            Assert.That(conn.FullState, Is.EqualTo(ConnectionState.Broken));
+        }
```

Comment:
If I see this correctly this is again a code cleanup that uses the modern using declaration but doesn't change any logic, which is highly appreciated but should best come in a separate PR.

Anyway, just leave it, because I'm probably too harsh once again and I probably couldn't hold up to the standards I'm propagating here myself.

Reply:
Oups, that was definitely not intended. Most likely, from when I was working with `TimeoutException` for sync and async queries.
While working on this pr, I tried to change as little as possible in existing codebase. But, in this case I think I will leave it as it is, so tests for sync and async timeouts in queries are looking the same.